src/pages/partners/payment-agent/_who-can-apply.tsx

```diff
@@ -70,7 +70,45 @@ const ButtonWrapper = styled(CenteredSection)`
     display: flex;
     justify-content: center;
 `
-
+const section_content = [
```

Comment:
Hi, I think we should use type to avoid adding new content with wrong types:
section_content:  SectionComponentProps[] = [

The same should be done for faq_schema in _faq-schema.ts